shell/platform/tizen/flutter_tizen_texture_registrar.cc

```diff
@@ -88,26 +98,36 @@ bool FlutterTizenTextureRegistrar::PopulateTexture(
 
 std::unique_ptr<ExternalTexture>
 FlutterTizenTextureRegistrar::CreateExternalTexture(
-    const FlutterDesktopTextureInfo* texture_info) {
+    const FlutterDesktopTextureInfo* texture_info,
+    FlutterDesktopRendererType renderer_type) {
   switch (texture_info->type) {
     case kFlutterDesktopPixelBufferTexture:
-      return std::make_unique<ExternalTexturePixelGL>(
-          texture_info->pixel_buffer_config.callback,
-          texture_info->pixel_buffer_config.user_data);
+      FT_UNIMPLEMENTED();
+      return nullptr;
       break;
```

Comment:
`break` should not be necessary after `return`.

Reply:
done thanks :+1: